integration-tests/test_collection.py

```diff
@@ -60,9 +134,24 @@ def test_output_specifying_both_dir_and_file(insights_client, tmp_path):
 
 
 def test_output_file_with_relative_path(insights_client):
-    """Test --output-file (with relative path)
-    Test --output-file option pointing to:
-        - relative path (so test will indicate path is already a directory)
+    """
+    :id: e1236f11-46c3-452c-81b8-3c61506e1841
+    :title: Test output File with Relative Path
+    :description:
+        Verifies that using a relative path that points to a directory
+        with the --output-file option fails with an appropriate error message
+    :tier: Tier 1
+    :steps:
+        1. Define a relative path pointing to a directory
+        2. Run insights-client with the --output file option poining to
+            this relative path
+        3. Verify that the command fails with a return code of 1
+        4. Verify the error message is as expected
```

Comment:
Again -- is "as expected" descriptive enough, or should this say that `The error message says that `--output` cannot be used with a directory?